src/helpers/figures/charts/pie_chart.ts

```diff
@@ -1,9 +1,9 @@
-import type {
-  BubbleDataPoint,
-  ChartConfiguration,
-  ChartDataset,
-  LegendOptions,
-  Point,
+import {
```

Comment:
seems like a useless diff. Maybe your ide/tooling automatically changed that. Could be worth investigating because it really seems useless and on my machine, I can revert it.